Tasks/NuGetInstallerV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -42,5 +42,6 @@
   "loc.messages.NGCommon_RemovingSources": "Preparing to set credentials in NuGet.config",
   "loc.messages.NGCommon_AddingSources": "Setting credentials in NuGet.config",
   "loc.messages.NGCommon_NoSourcesFoundInConfig": "No package sources were found in the NuGet.config file at %s",
-  "loc.messages.NGCommon_UnabletoDetectNuGetVersion": "Unknown NuGet version selected."
+  "loc.messages.NGCommon_UnabletoDetectNuGetVersion": "Unknown NuGet version selected.",
+  "loc.messages.DeprecatedTask": "This task is now deprecated. The task completed successfully, but builds are breaking. Functionality will stop on 01-16-2024. Please switch to using NuGetCommand@2's 'restore' option as soon as possible."
```

Comment:
Same comment, be less verbose, just describe right now what the state is and how to fix it

Reply:
Resolved!